src/main/java/mate/academy/bookstore/model/Book.java

```diff
@@ -34,6 +42,11 @@ public class Book {
     private String coverImage;
     @Column(nullable = false)
     private boolean isDeleted = false;
+    @ManyToMany(fetch = FetchType.LAZY)
```

Comment:
Could you please remove default fetch type? 

```suggestion
    @ManyToMany
```

Reply:
Logically))